addons/google_calendar/models/res_users.py

```diff
@@ -103,6 +103,18 @@ def _sync_google_calendar(self, calendar_service: GoogleCalendarService):
 
         return bool(events | synced_events) or bool(recurrences | synced_recurrences)
 
+    @api.model
+    def _get_admin_notification(self, failure_type, module_messages=None):
```

Comment:
Actually the goal here is to keep only base messages (mail__smtp_connection, mail__invalid_recipient, iap__low_credits, base__cron) from super() and since each override adds its related messages to the dict, no message will be lost.

Reply:
I don't understand, as you don't use module_messages ... each application should add its own keys inside the dict, unless I am mistaken somewhere.

Reply:
Yes but added to the shared ones from super() call ([messages are added to original ones here](https://github.com/odoo/odoo/blob/de138c2ca9bd0b325d6252108814eec84e0f463c/addons/mail/models/models.py#L166))